### PR TITLE
LRQA-53956 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-53956 
AppBuilder#SearchForDeployedItem

### CODE REVIEW

**Comment by alexcandidos** [comment]
ci:test:sf

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1167,6 +1167,58 @@ definition {
...
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "Third App",
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hi @nkkwon , after adding the 3 apps, I would suggest you:

Validation 4 (make sure the list of deployments are ordered by last modified date by default - most recente on top - you can look if the first item is Third App in the table, and First App is the last one in the table)

Then you perform the search by the term: "App" so you can cover the following validations: 
Validation 1 (search and retrieve **all** deployments that has a specific "term" - so we'll be testing "contains" search ).  
Validation 3 (search and see if the items are ordered by last modified date)
Additionally you can search by a unique term "First" as you were doing, so you can guarantee that the exact match search is also working.

Then, search by an invalid term:
Validation 5.

Then, clear the search results to do the following validations:
Validation 6 (make sure that all the deployments are shown again

Then, finish with Validation 2 (order the list of deployments by Name/Title;

**Comment by nkkwon** [comment]
@alexcandidos I added the suggestions that you had, I'm not able to do validation 2 in the end because it's still bugged https://issues.liferay.com/browse/LPS-106082

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase (hunk 1 above)
Comment (nkkwon) [reply modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Added the changes!

**Comment by alexcandidos** [comment]
Hi @nkkwon , thanks for the improvements and for remembered me about this bug! I believe that we could leave the validation 2 in the code and the test failing, it will force the team to fix it.

Could you please rebase your changes and send the PR again? So we can see it clear and then forward it to BChan.

Thanks!!

**Pull request closed without merging**